src/Orc.Controls.Tests/Orc.Controls.approved.cs

```diff
@@ -940,6 +946,15 @@ public interface IColorLegendItem : System.ComponentModel.INotifyPropertyChanged
         bool IsChecked { get; set; }
         bool IsSelected { get; set; }
     }
+    public static class IConfigurationServiceExtensions
+    {
+        public static System.DateTime? GetCalloutLastShown(this Catel.Configuration.IConfigurationService configurationService, Orc.Controls.ICallout callout) { }
```

Comment:
Is we saving all of this in the root configuration (configuration.xml in the %appdata%/appname?

Reply:
Yes. I didn't want a "CalloutConfigurationService". If we create such a service, there is no more need to put this whole control in Orc.Controls and should go into it's own library.